addons/mail/static/src/components/message/message.js

```diff
@@ -302,6 +304,14 @@ class Message extends Component {
                 if (value.new_value !== undefined){
                     value.new_value = value.new_value ? this.env._t("True") : this.env._t("False");
                 }
+            } else if (value.field_type === 'monetary' && value.currency_id) {
+                const currency_id = session.currencies[value.currency_id]
+                if (value.old_value !== false) {
+                    value.old_value = $.parseHTML(format.monetary(value.old_value, null, {currency: currency_id}))[0].textContent;
+                }
+                if (value.new_value !== false) {
+                    value.new_value = $.parseHTML(format.monetary(value.new_value, null, {currency: currency_id}))[0].textContent;
```

Comment:
We don't want to use jQuery, and we especially don't want to have to parse HTML to generate a string.

But I'm not sure to understand why it's even needed in this case? Could you clarify? The result of `formatMonetary` is not good to use as it is?

Reply:
The result of `format.monetary` is something like this `"$&nbsp;500.00"`, so instead of a space character in the message we will have `&nbsp;`

Reply:
Ha, I've just seen that there is a `forceString` option to avoid this